lib/util/versioning.js

```diff
@@ -55,6 +69,8 @@ function get_runtime_abi(runtime, target_version) {
     }
     if (runtime === 'node-webkit') {
         return get_node_webkit_abi(runtime, target_version || process.versions['node-webkit']);
+    } else if (runtime === 'electron') {
+        return get_electron_abi(runtime, target_version || process.versions.electron);
```

Comment:
what's the BKM for getting my runtime & target_version in, if I have a project that uses a package which consumes node-pre-gyp?